diff --git a/keras_cnn.py b/keras_cnn.py
index 879d732..ded8672 100644
--- a/keras_cnn.py
+++ b/keras_cnn.py
@@ -6,6 +6,7 @@ from keras.utils import np_utils
 from wandb.keras import WandbCallback
 import wandb
 import os
+import sys
 
 run = wandb.init()
 config = run.config
@@ -30,7 +31,8 @@ X_train = X_train.reshape(
     X_train.shape[0], config.img_width, config.img_height, 1)
 X_test = X_test.reshape(
     X_test.shape[0], config.img_width, config.img_height, 1)
-
+print(X_train.shape)
+sys.exit(0)
 # one hot encode outputs
 y_train = np_utils.to_categorical(y_train)
 y_test = np_utils.to_categorical(y_test)
